digits/config/tensorflow.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+import os
+import platform
+from subprocess import Popen, PIPE
+
+from . import option_list
+
+VARNAME_ENV_TFPY = 'TENSORFLOW_PYTHON'
+DEFAULT_PYTHON_EXE = 'python2'  # @TODO(tzaman) - use the python executable that was used to launch digits?
```

Comment:
@TimZaman we still want to do this, right?

@ethantang95 here's how we do it elsewhere:
https://github.com/NVIDIA/DIGITS/blob/v5.0.0/digits/dataset/tasks/analyze_db.py#L78